runtime/frequency/src/lib.rs

```diff
@@ -457,7 +457,6 @@ impl pallet_collective::Config<TechnicalCommitteeCollective> for Runtime {
 	type MaxProposals = TCMaxProposals;
 	type MaxMembers = TCMaxMembers;
 	type DefaultVote = pallet_collective::PrimeDefaultVote;
-	// TODO: this uses default but we don't have weights yet. Issue: #608
 	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
```

Comment:
And Line 448 as well:

```suggestion
	type WeightInfo = weights::pallet_collective::SubstrateWeight<Runtime>;
```

Reply:
This flips it over to use the new one instead of the built in one in the substrate crate.

Reply:
I applied your change suggestion.  Note that this issue was for the collective pallet (previous PR) while this PR is for the collator selection pallet.

Reply:
@rlaferla Ah so it is

- [ ] We need to fix the collective on line 448 as well.
- [ ] We currently have `type WeightInfo = ();` for the collator selection pallet which should reference the new weights.

Reply:
Fixed.